### PR TITLE
Blog post for Istio 2024-25 Steering Committee

### PR DESCRIPTION
**Pull request by craigbox**
This post announces the results of the Contribution Seat allocation, changes to the Community Seats as a result, and our intention to move the election timing to align our two cohorts.

### CODE REVIEW

**Comment by craigbox** [comment]
https://deploy-preview-14606--preliminary-istio.netlify.app/latest/blog/2024/steering-results/

File: content/en/blog/2024/steering-results/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Community seats
+
+As a result of this year's calculation, two of our Community Seat holders move to Contribution Seats. This creates two extra vacancies, which are allocated to the runners-up of our last election[^daocloud].
+
+We are pleased to welcome our two newest Community Seat holders, Mitch Connors from Aviatrix and Keith Mattix from Microsoft. Both are highly active maintainers and leaders in the project, and we are delighted to have them join the Steering Committee.
+
+## Proposed changes to election timing
+
+Our charter currently allocates Contribution Seats in February and holds the Community Seat election in July. We previously [anticipated a situation](https://github.com/istio/community/pull/609#issuecomment-877302152) where people would change seat type mid-term, and this has now come to pass.
```
Comment (ericvn) [inline content/en/blog/2024/steering-results/index.md:64]:
```suggestion
Our charter currently allocates Contribution Seats in February and holds the Community Seat election in July. We previously [anticipated a situation](https://github.com/istio/community/pull/609#issuecomment-877302152) where people would change seat types mid-term, and this has now come to pass.
```

File: content/en/blog/2024/steering-results/index.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+publishdate: 2024-02-15
+attribution: "Craig Box, for the Istio Steering Committee"
+keywords: [istio,steering,governance,community,election]
+---
+
+The Istio Steering Committee oversees the administrative aspects of the project, including governance, branding, marketing, and working with the CNCF.
+
+Every year, the leaders in the Istio project [estimate the proportion](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md) of the [hundreds of companies that have contributed to Istio in the past year](https://istio.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=contributions), and uses that metric to proportionally allocate nine Contribution Seats on our Steering Committee.
+
+Then, four Community Seats are voted for by our project members, with candidates being from companies that did not receive Contribution Seats.
```
Comment (ericvn) [inline content/en/blog/2024/steering-results/index.md:13]: Do we want to add that these 4 seats are 4 different companies?

File: content/en/blog/2024/steering-results/index.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+## Proposed changes to election timing
+
+Our charter currently allocates Contribution Seats in February and holds the Community Seat election in July. We previously [anticipated a situation](https://github.com/istio/community/pull/609#issuecomment-877302152) where people would change seat type mid-term, and this has now come to pass.
+
+We will therefore be voting on a change to our Charter which will move our Community Seat elections to February, to be held immediately after the allocation of Community Seats. It is our intention that the next annual election be held in February 2025.
+
+## The full group
+
+Following these changes, we have now have representation from nine companies:
```
Comment (ericvn) [inline content/en/blog/2024/steering-results/index.md:70]:
```suggestion
Following these changes, we now have representation from nine companies:
```

File: content/en/blog/2024/steering-results/index.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+Faseela K | Ericsson Software Technology | [kfaseela](https://github.com/kfaseela) | Community seat
+Kebe Liu | DaoCloud | [kebe7jun](https://github.com/kebe7jun) | Contribution seat
+Keith Mattix | Microsoft | [keithmattix](https://github.com/keithmattix) | Community seat
+Justin Pettit | Google | [justinpettit](https://github.com/justinpettit) | Contribution seat
+Lin Sun | Solo.io | [linsun](https://github.com/linsun) | Contribution seat
+Zhonghu Xu | Huawei | [hzxuzhonghu](https://github.com/hzxuzhonghu) | Contribution seat
+
+Our sincerest thanks to Ameer Abbas, April Kyle Nassi, Cale Rath and Chaomeng Zhang, whose terms have come to an end.
+
+The Steering Committee wishes to thank its members, old and new, and looks forward to working with them to continue to grow and improve Istio as a successful and sustainable open source project. We encourage everyone to get involved in the Istio community, contribute, vote, and help us shape the future of cloud native networking.
```
Comment (ericvn) [inline content/en/blog/2024/steering-results/index.md:90]: Working with them seems weird to me. Maybe:
```suggestion
The Steering Committee wishes to thank its members, old and new, and looks forward to continue to grow and improve Istio as a successful and sustainable open source project. We encourage everyone to get involved in the Istio community, by contributing, voting, and helping us shape the future of cloud native networking.
```

File: content/en/blog/2024/steering-results/index.md (hunk 2 above)
Comment (craigbox) [reply content/en/blog/2024/steering-results/index.md:13]: Maybe - that's always true by design, as you can only hold one Community Seat per company.

**Comment by craigbox** [comment]
Lint passing (grrr, #14609)

File: content/en/blog/2024/steering-results/index.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+As a result of this year's calculation, two of our Community Seat holders move to Contribution Seats. This creates two extra vacancies, which are allocated to the runners-up of our last election[^daocloud].
+
+We are pleased to welcome our two newest Community Seat holders, Mitch Connors from Aviatrix and Keith Mattix from Microsoft. Both are highly active maintainers and leaders in the project, and we are delighted to have them join the Steering Committee.
+
+## Proposed changes to election timing
+
+Our charter currently allocates Contribution Seats in February and holds the Community Seat election in July. We previously [anticipated a situation](https://github.com/istio/community/pull/609#issuecomment-877302152) where people would change seat type mid-term, and this has now come to pass.
+
+We will therefore be voting on a change to our Charter which will move our Community Seat elections to February, to be held immediately after the allocation of Community Seats. It is our intention that the next annual election be held in February 2025.
```
Comment (cetezadi) [inline content/en/blog/2024/steering-results/index.md:66]: This should say "to be held immediately after the allocation of _**Contribution**_ Seats" (emphasis mine)

Comment (craigbox) [reply content/en/blog/2024/steering-results/index.md:66]: Good catch, i put these in occasionally to see if anyone is reading closely 😁

**Review by ericvn — APPROVED** [review]

**Pull request merged**